recipe/meta.yaml

```diff
@@ -1,63 +1,119 @@
 {% set name = "awscli" %}
-{% set version = "1.27.7" %}
-{% set hash = "faa8acf66eea21447b04f0d021c4539cd714b2770f1b80cd2222ec68e3313c94" %}
-
-{% set vmajor,vminor,vpatch = version.split('.') %}
-# ALWAYS DOUBLE-CHECK THESE OFFSETS AGAINST UPSTREAM setup.py
-{% set botocore_major = vmajor | int + 0 %}
-{% set botocore_minor = vminor | int + 2 %}
-{% set botocore_patch = vpatch | int + 0 %}
-{% set botocore_pin = botocore_major ~ '.' ~ botocore_minor ~ '.' ~ botocore_patch %}
+{% set version = "2.8.12" %}
+{% set hash = "a1f16a8e0bb7fb098967f852b074f431ec4adbdfa51e714129945c67a9798537" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
+# v2 is not on PyPI https://github.com/aws/aws-cli/issues/6785
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/aws/aws-cli/archive/{{ version }}.tar.gz
   sha256: {{ hash }}
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv  # [not unix]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv && mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
+  # Build uses pep517 with a custom backend
+  # console_scripts entrypoints aren't used, so this can't be noarch
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv && mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
+  skip: true  # [py<38]
```

Comment:
3.7 was dropped as of https://github.com/conda-forge/awscli-feedstock/pull/1187, is this an upstream requirement or was it a build issue?

Reply:
[v2 is Python 3.8+ only](https://github.com/aws/aws-cli/blob/5da64c3bc8706756bc43d197a27ac0dba61c0e6e/pyproject.toml#L16). I didn't realise conda-forge had dropped Python 3.7, so I've removed this skip.